docs/inspiration/brand-guidelines/home-depot/index.qmd

```diff
@@ -19,7 +19,7 @@ color:
     orange: "#F96302"
     white: "#FFFFFF"
     black: "#000000"
-    warm-gray-ligt: "#F5F5F5"
+    warm-gray-light: "#F5F5F5"
     warm-gray-dark: "#747474"
     warm-gray-medium: "#C4C4C4"
   theme:
```

Comment:
this is also a problem, it should not have the `theme` field

```yaml
	foreground: black
  background: white
  primary: orange
  secondary: warm-gray-dark
  light: warm-gray-light
  dark: warm-gray-dark
```

Reply:
Ok! That fix is coming in the other PR.